app/javascript/react/components/ReviewTile.js

```diff
@@ -80,10 +80,49 @@ const ReviewTile = props =>{
     }
   }
 
+  const deleteReviewFetch = async () => {
+    try {
+      const response = await fetch(`/api/v1/sauces/${props.sauce.id}/reviews/${props.id}`,{
+        credentials: "same-origin",
+        method: "DELETE",
+        headers: {
+          "Accept": "application/json",
+          "Content-Type": "application/json"
+        },
+        body: JSON.stringify({user_id: props.reviewUserId})
+      }) 
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.status.text})`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      const sauceObject = await response.json()
+      props.setSauce(sauceObject.sauce)
```

Comment:
lets actually use setSauce to filter out the review that was deleted (but keeping the rest of sauce state)

Reply:
setting the props.sauce object in state and filtering out the deleted review from the reviews array in sauce